content/en/about/case-studies/bluecore/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Bluecore Leverages Istio to Migrate to Kubernetes"
+linkTitle: "Bluecore Leverages Istio to Migrate to Kubernetes"
+quote: "Istio enables an approach that allows you to start building right out of the box." 
+author:
+    name: "Shray Kumar"
+    image: "/img/authors/shray-kumar.jpg"
```

Comment:
Please crop the face square to match the aesthetic on the other pages.

Reply:
@craigbox Both changes done!